lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -265,8 +285,8 @@ private static function get_inner_blocks( $attributes, $block ) {
 		 * Allows modification of a navigation block menu items.
 		 *
 		 * @since 6.1.0
-		 *
 		 * @param \WP_Block_List $inner_blocks
+		 * @param int|null $navigation_post_id The navigation post ID.
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57754/commits/2de5bfda9595638e41b8cd72d28fedf694d4eebc